CONTRIBUTING.md

```diff
@@ -27,3 +42,38 @@ In order to accept your pull request, we need you to submit a CLA. You only need
 ## License
 
 By contributing to OCamlFormat, you agree that your contributions will be licensed under the LICENSE file in the root directory of this source tree.
+
+## Ocamlformat Labels
```

Comment:
I don't know what is best, but elsewhere ocamlformat is capitalized "OCamlFormat".